### PR TITLE
Apply dark theme to dl views, and new layout and icon

### PR DESCRIPTION
**Pull request by EdenSG**
image
image

@ArmandTrash requesting review of the style — does the "download on the app store" button work, or would you prefer a plain text button?

### CODE REVIEW

File: dl-snapchat.html [hunk 1]
```diff
@@ -3,18 +3,51 @@
...
+		</svg>
+		<p class="body-text">
+			Turn your Snapchat
+			<br />
+			stories into dreams.
+		</p>
+	</div>
+	<a href="https://thetrash.app.link/36xcKYfZr7" target="_blank">
+		<img class="off" src="/assets/images/appstorebutton_rainbow.svg" />
+		<!-- <img class="on" src="/assets/images/appstorebutton_white.svg" /> -->
```
Comment (chihyuC) [inline dl-snapchat.html:51]: maybe this can be enabled now that the bg is black again. it was looking awkward on a whilte bg.

Comment (EdenSG) [reply dl-snapchat.html:51]: sure, can do

Comment (EdenSG) [reply dl-snapchat.html:51]: at armand's request i'm gonna change these out for a button with "download trash"

**Comment by EdenSG** [comment]
new look: 

image

**Pull request merged**